service/ocean/providers/aws/cluster.go

```diff
@@ -2030,3 +2031,184 @@ func (o *Filters) SetVirtualizationTypes(v []string) *Filters {
 	}
 	return o
 }
+
+func (o *LaunchSpecification) SetClusterBlockDeviceMappings(v []*ClusterBlockDeviceMapping) *LaunchSpecification {
+	if o.BlockDeviceMappings = v; o.BlockDeviceMappings == nil {
+		o.nullFields = append(o.nullFields, "BlockDeviceMappings")
+	}
+	return o
+}
+
+type ClusterBlockDeviceMapping struct {
+	DeviceName  *string     `json:"deviceName,omitempty"`
+	NoDevice    *string     `json:"noDevice,omitempty"`
```

Comment:
NoDevice and VirtualName not available in the configuration in documentation

Reply:
removed unused parameters VirtualName and NoDevice